percona-xtradb-cluster/5.6/source/release-notes/Percona-XtraDB-Cluster-5.6.24-25.11.rst

```diff
@@ -0,0 +1,54 @@
+.. rn:: 5.6.24-25.11
+
+=======================================
+ |Percona XtraDB Cluster| 5.6.24-25.11 
+=======================================
+
+Percona is glad to announce the release of |Percona XtraDB Cluster| 5.6 on June 3rd 2015. Binaries are available from `downloads area <http://www.percona.com/downloads/Percona-XtraDB-Cluster-56/release-5.6.24-25.11/>`_ or from our :doc:`software repositories </installation>`.
+
+Based on `Percona Server 5.6.24-72.2 <http://www.percona.com/doc/percona-server/5.6/release-notes/Percona-Server-5.6.24-72.2.html>`_ including all the bug fixes in it, *Galera Replicator* 3.11 and on *Codership wsrep API* 25.11 is now the current **General Availability** release. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.6.24-25.11 milestone <https://launchpad.net/percona-xtradb-cluster/+milestone/5.6.24-25.11>`_ at Launchpad.
+
+New Features
+============
+
+ |Percona XtraDB Cluster| now allows reads in non-primary state by introducing a new session variable, :variable:`wsrep_dirty_reads`. This variable is boolean and is ``OFF`` by default. When set to ``ON``, a |Percona XtraDB Cluster| node accepts queries that only read, but not modify data even if the node is in the non-PRIM state (:bug:`1407770`, :bug:`1409618`).
+
```

Comment:
1409618 is a bit different in that I_S queries are allowed even without wsrep_dirty_reads set or wsrep_ready being 1. This is for existing monitoring applications. For more details chek https://github.com/percona/percona-xtradb-cluster/pull/18